README.md

```diff
@@ -198,22 +263,38 @@ Then deploy the `dist` folder
 
 ## Integration & Validation Tests
 
-For the integration tests to work, you have to launch the fake API and the dashboard first. Run the following commands in separate terminal sessions:
+For the integration tests to work, you have to first run the localstack application, launch the localAPI and serve the dashboard first. Run the following commands in separate terminal sessions:
 
+Run background localstack application.
 ```bash
-  $ node fake-api.js
-  $ APIROOT=http://localhost:5001 npm run serve
+  $ npm run start-localstack
 ```
 
-Run the test suite in another terminal:
+Serve the cumulus API (separate terminal)
+```bash
+  $ npm run serve-api
+```
 
+Serve the dashboard web application (another terminal)
 ```bash
-  $ npm run validation
+  $ npm run serve
+```
+
+If you're just testing dashboard code, you can generally run all of the above commands as a single docker-compose stack.
+```bash
+  $ npm run start-dashboard
+```
+This brings up LocalStack, Elasticsearch, the Cumulus localAPI, and the dashboard.
+
+Run the test suite (yet another terminal window)
+```bash
+  $ npm run validate
   $ npm run cypress
```

Comment:
And add the ES Host and LOCALHOST to the command, so the user doesn't have to deal with it

Reply:
8a0bde66